src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryEntityDAO.groovy

```diff
@@ -190,6 +190,23 @@ class DirectoryEntityDAO {
     }
 
     /**
+     * Gets an osuuid as a long
+     *
+     * @param ldapEntry
+     * @return Long osuuid
+     */
+    private static Long getOSUUid(LdapEntry ldapEntry) {
+        String osuuidString = get(ldapEntry, 'osuuid')
+        Long osuuid
```

Comment:
How about if we set a default value of 0 or something else so that we don't have a null? Sometimes the test data makes it into production and the quality of data isn't always good.